modules/android/pages/vector-search.adoc

```diff
@@ -122,9 +181,19 @@ This metric focuses on the alignment of two vectors, the similarity of direction
 Only the direction of the vectors matter.
 The smaller the distance value, the more similar the vectors are.
 
+== Hybrid Vector Search
+
+You can use Vector Search with regular SQL++ query language. 
+Vector search will be performed on the documents that have been filtered based on the criteria specified in the WHERE clause.
+No LIMIT clause is required for Hybrid Vector Search.
+
+See the https://www.couchbase.com/blog/hybrid-search/[Hybrid Search blog post] for more information about Hybrid Search.
+
+IMPORTANT: You cannot `OR` with Hybrid Vector Search or when using `APPROX_VECTOR_DISTANCE()` with other expressions in the `WHERE` clause.
+
 == Next Steps
 
-Now that you're familiar with the core concepts of vector search, including vector embeddings, indexing, encoding, centroids, dimensions, and distance metrics, it's time to put your knowledge into practice and start building AI applications.
+Now that you're familiar with the core concepts of vector search, it's time to put your knowledge into practice and start building AI applications.
```

Comment:
I would strike the Next Steps section entirely. I don't think it adds value.

Reply:
Done